releases/release-1.29/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Kubernetes 1.29 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.29
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.29/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.29/release-team.md)
+* [Meeting Minutes](https://bit.ly/k8s129-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.29 Release Calendar][k8s129-calendar]
+* Contact: [#sig-release] on slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Board](https://bit.ly/k8s129-enhancements)
+* [Feature blog Tracking Board](http://bit.ly/k8s129-feature-blog)
+* [Bug Triage Tracking Board](https://bit.ly/k8s129-bug-triage-tracking)
+* [CI Signal Project Board & Issue Tracking](#)
+* [Retrospective Document](https://bit.ly/k8s129-retro)
+* [kubernetes/sig-release v1.29 milestone](https://github.com/kubernetes/kubernetes/milestone/62)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## Summary
+
+The 1.29 release cycle is as follows:
+
+- **Monday 4th September 2023**: Week 1 — Release cycle begins
+- **Thursday 28th September 2023**: Week 4 — [Production Readiness Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
+- **[01:00 UTC Friday 6th October 2023 / 18:00 PDT Thursday 5th October 2023](https://everytimezone.com/s/829cf85c)**: Week 5 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Monday 6th - Thursday 9th November 2023**: Week 10 - [KubeCon NA](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)
+- **[01:00 UTC Wednesday 15th November 2023 / 18:00 PDT Tuesday 14th November 2023](https://everytimezone.com/s/b708fa7a)**: Week 11 — [Code Freeze](../release_phases.md#code-freeze)
```

Comment:
For reference, 1.26 had code freeze on Nov 8th and release on Dec 9th, roughly one month apart (knowing we lose a ~half week to holidays for many contributors at the end of Nov).

This schedule puts code freeze a week later (Nov 15th) and release four days earlier (Dec 5th). That short of a time period will be difficult to get enough release candidates out and get useful feedback around US holidays.

Also, the week before code freeze is typically extremely busy for reviewers/approvers. This schedule puts code freeze immediately after kubecon, which will steal time from many reviewers/approvers. Can we move code freeze *before* kubecon?

Reply:
@liggitt, I've moved the Code Freeze to Week 9 (Nov 1st) before KubeCon NA 2023.

With this change, **`rc.0` cut** (Nov 14, week 11) and **`rc.1` cut** (Nov 28, week 13) are now two weeks apart.

Will two weeks time (including US holiday week) for collecting `rc.0` feedback  would be ok?

Reply:
that seems good to me, would welcome feedback from others

Reply:
Sounds good to me, too.

Reply:
Enhancements freeze is Oct 5 and code freeze is Oct 31.  This doesn't seem like enough time to me.

Reply:
> Enhancements freeze is Oct 5 and code freeze is Oct 31. This doesn't seem like enough time to me

Hi @enj, I agree, ***but*** we're in a bit of a time crunch due to KubeCon falling on Week 10 of the 1.29 Release cycle, plus the US holiday weeks following that.

I did think about pushing the Code Freeze to Week 11 (fe63e79be581e3b26655b1d30d49de76dfb226b0), but after weighing everything, it doesn't seem like the best idea considering all the following reasons we discussed earlier:
- it could overload reviewers/approvers during kubeCon,
- high chance that reviewers/approvers might be unavailable pre-Code Freeze,
- plus a major US holiday week between `rc.0` and `rc.1`, which means very limited feedback.